website/src/data/landingPageData.js

```diff
@@ -0,0 +1,63 @@
+export const featureCards = [
+  {
+    title: "Direct Vulnerability Patching",
+    description: "Patches container images instantly without requiring full rebuilds - just adds a lightweight patch layer on top of existing images.",
+    icon: "/img/feature-patching.png",
+  },
+  {
+    title: "Multi-Package Manager Support",
+    description: "Supports APK, DPKG, and RPM package managers, covering a wide range of popular base images like Alpine, Debian, Ubuntu, and CBL-Mariner.",
+    icon: "/img/feature-pkg-manager.png",
+  },
+  {
+    title: "Multi-Platform & Distroless Support",
+    description: "Builds multi-architecture images and is capable of patching minimal 'distroless' images that lack shells or package managers.",
+    icon: "/img/feature-multi-platform.png",
+  },
+  {
+    title: "Ecosystem & Scanner Compatible",
+    description: "Integrates with popular vulnerability scanners like Trivy and can be used in any CI/CD pipeline to automate your patching workflow.",
+    icon: "/img/feature-ecosystem.png",
+  },
+];
+
+export const adopters = [
```

Comment:
can we add descriptions for these? similar to "Looking for a managed service or integration?" section https://open-policy-agent.github.io/gatekeeper/website/